pennylane/devices/tests/test_gates.py

```diff
@@ -144,6 +145,7 @@
 S = np.diag([1, 1j])
 T = np.diag([1, np.exp(1j * np.pi / 4)])
 SX = 0.5 * np.array([[1 + 1j, 1 - 1j], [1 - 1j, 1 + 1j]])
+V = 1 / sqrt(2) * np.array([[1, -1j], [-1j, 1]], dtype=complex)
```

Comment:
Why choose V to be this? This V propto I - 1j X, square of which should be propto jX and is not what you claimed in the description.

Reply:
Oh it seems this PR already closed due to duplication. I'll just resolve this conversation